CMS/cms-basics

```diff
@@ -0,0 +1,345 @@
+|Data export	|Yes	|Yes	|
+|Data import |No |Yes |
+|Who can access	|Collaborators of your app on Skygear	|Admin users in Skygear user role record	|
+|Customizable	|No. It default shows all tables and columns.	|Yes, with config in YML format.	|
+
+
+## Enable CMS plugin at Skygear Portal
+
+By default, CMS is not enabled on the portal. You can turn on the the CMS plugin at `Skygear Portal > CMS. `If you are not a new Skygear users and wanted to use this new version of CMS, re-enable the CMS plug-in 🤓
+
+## CMS configuration overview
+
+Skygear supports a simple way to configure CMS via editing the CMS Configuration YML. You can find the current config YML at ` Skygear Portal > CMS > CMS Configuration`.
+
+The YML itself looks explanatory, still there are some details we want you to know. Below is a simple overview of what you can configure in the CMS.
+
+```
+site:
+  // Pages in the CMS. Available pages will be shown on the left sidebar.
+
+records:
+  user:
+    // Configuring the user record
+    list:
+      //Displayable fields on the first page
+      fiters:
+        //Filtable fields on the first page
+    show:
+      //Displayable fields in the detial page
+    edit:
+      //Editable fields
+    new:
+      //Fill-in-able fields when creating a new record
+```
+
+## Example: configuring a basic CMS
+
+In the following examples, we will be using the below database structure as a sample.
+
+|Record	|Fields	|	|	|	|	|
+|---	|---	|---	|---	|---	|---
+|user	|_id	|_created_at	|username	|role	|avatar	|
+|blogpost	|_id	|_created_at	|title	|content	|
+|comment	|_id	|_created_at	|comment	|blogpostId (Skygear reference)	|
+
+### Site
+
+**Site** defines what pages to be shown in the CMS. All availabe pages will be shown on the left sidebar.
+
+There are 3 types of pages you can show:
+
+1. Record: a page to display all the records of a record type.
+2. Push Notifications: a page to broadcast push messages to your users. (Coming soon)
+3. User management: a page to manage your users, from creating a new user account, to blocking problemtics users. (Coming soon)
+
+For example, if you want to display all the records from **user** and the **blogpost**, you should write in the YML:
+
+```
+site:
+  - type: Record
+    name: user
+    label: User
+  - type: Record
+    name: blogpost
+    label: Blogpost
+```
+
+`name` refers to the record type in the database. `label` refers to the name to be shown on the CMS.
+
+If you only have 'Site' in the YML config, you will only see a blank page in the CMS because we haven't defined what to display on the pages. Let see how to do it below  👇
+
+
+### Records
+
+By now we have already defined two pages - **User** & **Blogpost**. Next we should define what to show in these pages.
+
+In general there are **FOUR** views you can customize: **list, show, edit and new**.
+
+* **List**: displayable fields on the first page. you can further set which fields can be **filtered**
```

Comment:
fields display as columns on the listing of records, you can further customize which fields can be **filtered**